### ISSUE TITLE
use kubectl_jll to provide kubectl dependency

### ISSUE THREAD

**Issue by jrevels** [beacon-biosignals/K8sClusterManagers.jl#6]
https://github.com/JuliaPackaging/Yggdrasil/pull/2247

**Comment by kolia** [comment]
`kubectl_jll` has been merged

https://github.com/JuliaPackaging/Yggdrasil/pull/2247

**Comment by ericphanson** [comment]
The JLL package has been created and registered, so we can now add `kubectl_jll` as a dependency and do e.g.

```julia
julia> using kubectl_jll

julia> kubectl() do exe
           run(`$exe version --client`)
       end
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.0", GitCommit:"af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38", GitTreeState:"clean", BuildDate:"2020-12-08T17:59:43Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Process(`/home/ec2-user/.julia/artifacts/94f44147f5fc89510556d9afafc210a47d00eba4/bin/kubectl version --client`, ProcessExited(0))
```

**Comment by omus** [comment]
Closed by #10
